net-mqtt.cabal

```diff
@@ -5,7 +5,7 @@ cabal-version: 1.12
 -- see: https://github.com/sol/hpack
 
 name:           net-mqtt
-version:        0.8.1.1
+version:        0.8.2.0
```

Comment:
Usually version numbers are assigned by the author when a new version is released and should not be changed in a pull request.